src/main/resources/reports/incident-count.json

```diff
@@ -16,14 +16,38 @@
   ],
   "policy": [
     {
-    "id": "role-based-policy",
-    "type": "access",
-    "rule": [
-      {
-        "effect": "permit",
-        "condition": []
-      }
-    ]
+      "id": "access",
+      "type": "access",
+      "rule": [
+        {
+          "effect": "permit",
+          "condition": [
+            {
+              "match": [
+                "${role}",
+                "ROLE_INCIDENT_REPORTS__RO",
```

Comment:
Protecting the data via a role.  Its questionable whether both the API endpoint AND the call need this restriction, maybe its only needed here, its just I hate "open" API endpoints and HMPPS dev community have agreed we shouldn't generally do that for any data other than reference data